jekyll/assets/css/navbar.css

```diff
@@ -26,7 +26,6 @@
   .nav-toggle.two .widescreen a.logo {
     position: absolute;
     top: 18px;
-    left: 0;
```

Comment:
The left nav is now out of alignment (too far left) with the search bar and logo:
![Screen Shot 2021-03-02 at 4 25 54 PM](https://user-images.githubusercontent.com/5113432/109721189-092aef00-7b79-11eb-9fcc-3f49b08f0ab4.png)